### PR TITLE
issue: 770746 Add jenkins check support on github pull request

### PR DESCRIPTION
**Pull request by igor-ivanov**
Added basic verification capabilities
- rpm
- build gcc/icc
- cov

valgrind will be later
style is disabled (should be added later)

@Di0gen could you look at this PR

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/31/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
report icc failure

File: contrib/jenkins_tests/test.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+test_tap=${WORKSPACE}/${prefix}/test.tap
+echo "1..1" > $test_tap
+ 
+cd $test_dir
+eval "LD_PRELOAD=$test_lib $test_app sr --tcp  > /dev/null 2>&1 &"
+sleep 5
+
+eval "LD_PRELOAD=$test_lib $test_app pp --tcp -m202 -t5"
+echo "ok 1 simple" > $test_tap
+
```
Comment (OphirMunk) [inline contrib/jenkins_tests/test.sh]: Do you check sockperf PP results? Need to verify that:
* No failure occurred
* Min/Avg/Max latencies are within the expected norm.
* Would be interesting to compare to previous commit results and verify no performance degradation.

Anyway an official spec about VMA acceptance tests is under preparation. Will see how to integrate it into Jenkins

**Comment by Di0gen** [comment]
+1 
Looks good, Ophir, let's merge those changes to repository, it allow us be up jenkins tests.

File: contrib/jenkins_tests/test.sh (hunk 1 above)
Comment (igor-ivanov) [inline contrib/jenkins_tests/test.sh]: @OphirMunk This test is very simple and does not support any failure check. We should extend one. I think about using sockperf/verifier. What do you think?

Comment (OphirMunk) [inline contrib/jenkins_tests/test.sh]: OK. 
We will add a set of sockperf tests with further commits.

**Pull request merged**